src/styles/blocks/calendar.scss

```
@@ -0,0 +1,50 @@
+@import "./utils/variables";
+@import "../utils/functions";
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+
+  gap: $margin;
+  width: widthContainer($cell-size);
+
+  @for $day from 28 through 31 {
+    &--length-#{$day}>:nth-child(n + #{1+$day}) {
+      display: none;
+    }
+  }
+
+  &__day {
+    width: $cell-size;
+    height: $cell-size;
+
+    border: 1px solid #000;
```

don't forget to create variables for all colors
```suggestion
    border: 1px solid $c-black;
```